apps/publications/src/assets/i18n/en.json

```
@@ -100,15 +98,17 @@
     "PREV": "Back",
     "FINISH": "Finish",
     "CREATE": "Create publication",
-    "RETURN": "Return",
+    "CANCEL": "Cancel",
     "SIMILAR_FOUND": "Similar publication already exists",
     "CHECK_BELOW": "Please check if your publication is listed below",
     "NOT_BELOW": "If NO, continue with Create button below.",
-    "IS_BELOW": "If YES, click on it to see details and add yourself as an author. Afterwards continue with the Return button. If desired publication is locked for changes, notify administrators about your request.",
+    "IS_BELOW": "If YES, click on it to see details and add yourself as an author. Afterwards continue with the Cancel button. If desired publication is locked for changes, notify administrators about your request.",
     "CHECK_PERFORM": "Checking for similar publications",
     "NO_SIMILAR": "Publication similarity check passed",
     "SUCCESS": "Publication was successfully created.",
-    "PUB_CREATED": "Publication was created, however it is not fully set up. You can finish setting up your publication by completing all steps. Additionally you can always edit publications on their detail."
+    "PUB_CREATED": "Publication was created, however it is not fully set up. You can finish setting up your publication by completing all steps. Additionally you can always edit publications on their detail.",
+    "NOT_AN_AUTHOR_DIALOG_TITLE": "You are not set as an author",
```

Just so it's not the same as start of the alert. Maybe you can think of even better title.
```suggestion
    "NOT_AN_AUTHOR_DIALOG_TITLE": "Not an author",
```